test/performance/parallel_scenarios.cpp

```diff
@@ -37,7 +37,7 @@ int main(int argc, char **argv) {
 
         auto bench_one = [&]() {
             auto t1 = std::chrono::high_resolution_clock::now();
-            callable(i, o, memory_limit, in, out);
+            (void)callable(i, o, memory_limit, in, out);
```

Comment:
This actually returns an error code doesn't it? Might be worth checking it outside the timing scope.

Reply:
Added comment